lisa/schema.py

```diff
@@ -1487,6 +1487,8 @@ class TestCase(BaseTestCaseFilter):
             field_function=fields.Int, validate=validate.Range(min=0)
         ),
     )
+    # each test case will have this timeout, default is 3600 seconds
+    timeout: Optional[int] = 3600
```

Comment:
It's not optional by default, please set the default value to `None`. So the test case default timeout are respected.

Reply:
I have set the default value to 0 to maintain the consistency with the other places in other repo. Please do let me know if you feel otherwise.